prefect_airbyte/connections.py

```diff
@@ -45,21 +44,28 @@ async def trigger_sync(
     when it receives an error status code from an API call.
 
     Args:
-        str airbyte_server_host : Hostname of Airbyte server where connection is
-            configured.
-        str airbyte_server_port: Port that the Airbyte server is listening on.
-
-        str airbyte_api_version: Version of Airbyte API to use to trigger connection
-            sync.
-        str connection_id: the Airbyte connection ID
-        int poll_interval_s: how often to poll the
-            Airbyte API for sync status, if provided this will
-            override the default polling time of 15 seconds.
-        bool status_updates: whether to log status as the task polls jobs
-        str timeout: The request `timeout` for the `httpx.AsyncClient`
+        connection_id (str): the Airbyte connection ID to trigger a sync for.
```

Comment:
The types in the docstrings shouldn't be necessary. The docstring parser that we use is able to pull the type from the type hints on the function signature.
```suggestion
        connection_id: the Airbyte connection ID to trigger a sync for.
```